tests/Elasticsearch/Tests/ClientBuilder/DummyLogger.php

```diff
@@ -19,5 +19,5 @@
 
 class DummyLogger
 {
-    
+
```

Comment:
Fixed in https://github.com/elastic/elasticsearch-php/pull/1068

Reply:
when your PR will be merged I'll resolve conflict.